starport/cmd/scaffold.go

```diff
@@ -86,6 +87,7 @@ func scaffoldType(
 
 	sm, err := sc.AddType(cmd.Context(), typeName, placeholder.New(), kind, options...)
 	if err != nil {
+		checkVersion(appPath)
```

Comment:
AFAIR we've decided to show the warning in any terms when there is version mismatch instead of only printing on scaffolding errors. cc @fadeev

Reply:
Yes, let's print the warning every time a user scaffolds something and there is a version mismatch.

Reply:
https://github.com/tendermint/starport/pull/1550/commits/6a1cfe81ecade2913d0a96a4de34c3a9c78ffc58